addons/reorder-custom-inputs/userscript.js

```diff
@@ -0,0 +1,73 @@
+export default async function ({ addon, console }) {
+  const Blockly = await addon.tab.traps.getBlockly();
+
+  function createArrow(direction, callback) {
+    const path = direction === "left" ? "M 17 13 L 9 21 L 17 30" : "M 9 13 L 17 21 L 9 30";
+
+    Blockly.WidgetDiv.DIV.insertAdjacentHTML(
+      "beforeend",
+      `
+            <svg width="20px" height="40px" 
+                 style="left: ${direction === "left" ? "calc(50% - 20px)" : "calc(50% + 20px)"}" 
+                 class="blocklyTextShiftArrow">
+                <path d="${path}" fill="none" stroke="#FF661A" stroke-width="2"></path>
+            </svg>`
+    );
```

Comment:
I'm not sure if doing this is allowed in addons (as I know `innerHTML` isn't, and this similarly involves adding HTML from a string).

Reply:
The innerHTML thing is to prevent css attacks however here It is allowed because it doesn't take in user provided data and add it to the page. It's no different to just adding an svg file.